src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -26,7 +28,17 @@
 DEFINE_string(redis_address, "", "The ip address of redis.");
 DEFINE_bool(redis_enable_ssl, false, "Use tls/ssl in redis connection.");
 DEFINE_int32(redis_port, -1, "The port of redis.");
-DEFINE_string(log_dir, "", "The path of the dir where log files are created.");
+DEFINE_string(event_log_dir,
+              "",
+              "The path of the dir where event log files are created.");
+DEFINE_string(
+    ray_log_stdout_filepath,
+    "",
+    "The filename to dump gcs server log on stdout, which is written via `RAY_LOG`.");
+DEFINE_string(
+    ray_log_stderr_filepath,
+    "",
+    "The filename to dump gcs server log on stderr, which is written via `RAY_LOG`.");
```

Comment:
Why do we need both stdout and stderr? All logs go to one place?

Reply:
Because current logic for spdlog controls both stdout and stderr: https://github.com/ray-project/ray/blob/f54efbb6f9d6f1c6e5802c5c5af5e4fd5d3ce015/src/ray/util/logging.cc#L380-L391